test/sql-tap/gh-6024-funcs-return-bin.c

```diff
@@ -0,0 +1,61 @@
+#include "msgpuck.h"
+#include "module.h"
+#include "uuid/mp_uuid.h"
+#include "mp_decimal.h"
+
+enum {
+	BUF_SIZE = 512,
+};
+
+int
+ret_bin(box_function_ctx_t *ctx, const char *args, const char *args_end)
```

Comment:
I'd better (void) args and (void) args_end to silence probable compiler warnings (unused variable)

Reply:
Fixed.